samples/bot-formatting-cards/csharp/README.md

```diff
@@ -0,0 +1,158 @@
+---
+page_type: sample
+description: This sample demonstrates the use of different formatting on cards
```

Comment:
This sample shows how to use different formatting on cards.

Reply:
Fixed